opencdx-client/src/main/java/cdx/opencdx/client/service/OpenCDXAnalysisEngineClient.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2024 Safe Health Systems, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package cdx.opencdx.client.service;
+
+import cdx.opencdx.client.dto.OpenCDXCallCredentials;
+import cdx.opencdx.client.exceptions.OpenCDXClientException;
+import cdx.opencdx.grpc.service.health.*;
+
+/**
+ * Interface for communicating with the Temperature service.
+ */
+public interface OpenCDXAnalysisEngineClient {
```

Comment:
opencdx-client/src/main/java/cdx/opencdx/client/config/ClientConfig.java  requires changes to support the new client.